main.smd

```diff
@@ -18,7 +24,7 @@ title: Stencila
 ````
 
 ````html raw
-<div class="flex items-center justify-end gap-x-4">
+<div class="flex items-center justify-end gap-x-4 pr-12">
   <a href="https://github.com/stencila/stencila#readme" class="header-link"><img  src="images/github.svg" alt="github" class="w-6"/></a>
   <a href="https://discord.gg/GADr6Jv" class="header-link"><img  src="images/discord.svg" alt="discord" class="w-6"/></a>
 </div>
```

Comment:
I think these 'header-link' classes can be removed? unless they're used in the stencila theme stylesheet.